### PR TITLE
changed size and adjusted color of delete buttonon modal edit asset b…

### PR DESCRIPTION
**Pull request by dlangston**
…utton

### CODE REVIEW

**Comment by dlangston** [comment]
@DanielJMaher @birdage

**Comment by DanielJMaher** [comment]
@birdage

File: ooiui/static/css/custom/custom.css [hunk 1]
```diff
@@ -107,6 +107,14 @@ li {
   background-color: #4fd934;
   color: #FFF;
 	line-height: .9;
+
+  border-radius: 5px;
+  color: #004773;
+  font-size: 14px;
+	font-weight: 300;
```
Comment (birdage) [inline ooiui/static/css/custom/custom.css]: @dlangston fix indentation please, make sure your using spaces not tabs

**Comment by DanielJMaher** [comment]
@dlangston Please fix

**Comment by dlangston** [comment]
@DanielJMaher @birdage fixed indentation

**Pull request merged**